core/model-vocabulary/src/main/java/org/eclipse/rdf4j/model/vocabulary/GEO.java

```diff
@@ -11,25 +11,538 @@
+	 */
+	public static final IRI DIMENSION;
+
+	/**
+	 * The geo:hasMetricArea property
+	 *
+	 * @see <a href="http://www.opengis.net/ont/geosparql#hasMetricArea">The geo:hasMetricArea property</a>
+	 */
+	public static final IRI HAS_METRIC_AREA;
+
+	/**
+	 * The geo:hasMetricLength property
+	 *
+	 * @see <a href="http://www.opengis.net/ont/geosparql#hasMetricLength">The geo:hasMetricLength property</a>
+	 */
+	public static final IRI HAS_METRIC_LENGTH;
+
+	/**
+	 * The geo:hasMetricPerimeterLength property
+	 *
+	 * @see <a href="http://www.opengis.net/ont/geosparql#hasMetricPerimeterLength">The geo:hasMetricPerimeterLength
+	 *      property</a>
+	 */
+	public static final IRI HAS_METRIC_PERIMETER_LENGTH;
+
+	/**
+	 * The geo:hasMetricSize property
+	 *
+	 * @see <a href="http://www.opengis.net/ont/geosparql#hasMetricSize">The geo:hasMetricSize property</a>
+	 */
+	public static final IRI HAS_METRIC_SIZE;
+
+	/**
+	 * The geo:hasMetricSpatialAccuracy property
+	 *
+	 * @see <a href="http://www.opengis.net/ont/geosparql#hasMetricSpatialAccuracyT">The geo:hasMetricSpatialAccuracyT
+	 *      property</a>
+	 */
+	public static final IRI HAS_METRIC_SPATIAL_ACCURACY;
+
+	/**
+	 * The geo:hasMetricSpatialResolution property
+	 *
+	 * @see <a href="http://www.opengis.net/ont/geosparql#hasMetricSpatialResolution">The geo:hasMetricSpatialResolution
+	 *      property</a>
+	 */
+	public static final IRI HAS_METRIC_SPATIAL_RESOLUTION;
+
+	/**
+	 * The geo:hasMetricVolume property
+	 *
+	 * @see <a href="http://www.opengis.net/ont/geosparql#hasMetricVolume">The geo:hasMetricVolume property</a>
+	 */
+	public static final IRI HAS_METRIC_VOLUME;
+
+	/**
+	 * The geo:hasSerialization property
+	 *
+	 * @see <a href="http://www.opengis.net/ont/geosparql#hasSerialization">The geo:hasSerialization property</a>
+	 */
+	public static final IRI HAS_SERIALIZATION;
+
+	/**
+	 * The geo:isEmpty property
+	 *
+	 * @see <a href="http://www.opengis.net/ont/geosparql#isEmpty">The geo:isEmpty property</a>
+	 */
+	public static final IRI IS_EMPTY;
+
+	/**
+	 * The geo:isSimple property
+	 *
+	 * @see <a href="http://www.opengis.net/ont/geosparql#isSimple">The geo:isSimple property</a>
+	 */
+	public static final IRI IS_SIMPLE;
+
+	/**
+	 * The geo:spatialDimension property
+	 *
+	 * @see <a href="http://www.opengis.net/ont/geosparql#spatialDimension">The geo:spatialDimension property</a>
+	 */
+	public static final IRI SPATIAL_DIMENSION;
+
+	/**
+	 * The geo:wktLiteral property
+	 *
+	 * @see <a href="http://www.opengis.net/ont/geosparql#wktLiteral">The geo:wktLiteral property</a>
+	 */
 	public static final IRI WKT_LITERAL;
 
 	public static final String DEFAULT_SRID = "http://www.opengis.net/def/crs/OGC/1.3/CRS84";
 
 	static {
+
+		// classes
+		FEATURE = Vocabularies.createIRI(NAMESPACE, "Feature");
+		FEATURE_COLLECTION = Vocabularies.createIRI(NAMESPACE, "FeatureCollection");
```

Comment:
I've also stopped using all caps for some of the newer vocabularies. I know it makes things a bit ugly when one vocabulary uses all caps and another doesn't, but it's just so much easier to understand what it what when using the exact same name for the variable in Java and in RDF.

@JervenBolleman Maybe you can chime in with your opinion?

Reply:
I agree with trying to use the same "spelling" as much as possible for constants. I had started a script to make it consistent for all (Deprecating the UPPER_SNAKE_CASE) but I need to look for this and see if I can finish it.

Reply:
done. Please note that I leave the following constant properties as it is:
- `geo:asWKT`
- `geo:wktLiteral`